gossip/src/crds_value.rs

```diff
@@ -17,10 +19,12 @@ use {
 
 /// CrdsValue that is replicated across the cluster
 #[cfg_attr(feature = "frozen-abi", derive(AbiExample))]
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq)]
+#[derive(Serialize, Clone, Debug, PartialEq, Eq)]
 pub struct CrdsValue {
     signature: Signature,
     data: CrdsData,
+    #[serde(skip_serializing)]
+    hash: Hash, // Sha256 hash of [signature, data].
```

Comment:
ok correct me if I am wrong, but when we go to serialize a `CrdsValue` and send it, only the `signature` and `data` get serialized and sent, right? We don't actually send the hash between nodes. Upon reception of the `CrdsValue`, the hash is calculated and populated when the `CrdsValue` is deserialized.

Reply:
right